dxr/plugins/xpidl/tests/test_menus_and_filters/code/nsIDownloadHistory.idl

```diff
@@ -0,0 +1,65 @@
+/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*-
```

Comment:
How do you feel about the size and sharedness of this test harness? Generally, except for smoke tests, I've tried to keep test cases minimal and unshared, so we know we aren't invalidating tests if we have to change the code later.